src/App.jsx

```diff
@@ -1,107 +1,30 @@
 import './App.scss';
 
-// import postsFromServer from './api/posts.json';
-// import commentsFromServer from './api/comments.json';
-// import usersFromServer from './api/users.json';
+import postsFromServer from './api/posts.json';
+import commentsFromServer from './api/comments.json';
+import usersFromServer from './api/users.json';
+import { PostList } from './components/PostList';
+
+function getUserById(userId) {
+  return usersFromServer.find(user => user.id === userId)
+    || null;
+}
+
+function getCommentByPostId(postId) {
+  return commentsFromServer.filter(comment => comment.postId === postId)
+    || null;
+}
```

Comment:
null is redudant because filter always return an array
```suggestion
function getCommentByPostId(postId) {
  return commentsFromServer.filter(comment => comment.postId === postId);
}
```